src/mono/wasm/debugger/DebuggerTestSuite/AsyncTests.cs

```diff
@@ -0,0 +1,104 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.WebAssembly.Diagnostics;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace DebuggerTests
+{
+    public class AsyncTests : DebuggerTestBase
+    {
+
+        // FIXME: method with multiple async blocks - so that we have two separate classes for that method!
+        // FIXME: nested blocks
+        // FIXME: Confirm the actual bp location
+        // FIXME: check object properties..
+
+        //FIXME: function name
+        [Fact] // Same as Instance case
+        public async Task AsyncLocalsInContinueWithStaticBlock() => await CheckInspectLocalsAtBreakpointSite(
```

Comment:
This can be combined with `AsyncLocalsInContinueWithInstanceBlock`. They are identical, except for the method name in two places.

Reply:
done.